gammapy/image/radial_profile.py

```diff
@@ -0,0 +1,87 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Image utility functions"""
+import logging
+import numpy as np
+from astropy.coordinates import Angle
+from astropy.wcs.utils import skycoord_to_pixel
+from astropy.table import QTable
+from scipy.ndimage.measurements import labeled_comprehension
+
+__all__ = [
+    'label_image',
+    'radial_profile',
+]
+log = logging.getLogger(__name__)
+
+
+def label_image(image, theta_bin=None, center=None):
```

Comment:
Rename `label_image` to `radial_profile_label_image` (since it's specifically creating the label image for radial profile computation).